ui/e2e/src/tests/markdown.spec.ts

```diff
@@ -0,0 +1,49 @@
+// Copyright 2023 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import happoPlaywright from 'happo-playwright';
+import { test } from '../fixtures/dashboardTest';
+
+test.use({
+  dashboardName: 'MarkdownPanel',
+});
+
+test.describe('Dashboard: Markdown Panel', () => {
+  test.beforeEach(async ({ context }) => {
+    await happoPlaywright.init(context);
+  });
+
+  test.afterEach(async () => {
+    await happoPlaywright.finish();
+  });
+
+  ['Headings', 'Text', 'Links', 'Code', 'Lists', 'Tables'].forEach((name) => {
+    test(`displays ${name} as expected`, async ({ page, dashboardPage }) => {
```

Comment:
Learning about Playwright: What is difference between `page` and `dashboardPage`?

Reply:
`page` is a [page](https://playwright.dev/docs/api/class-page) in the playwright API that allows you to interact with a tab in the browser during testing. 

`dashboardPage` is an instance of the [`DashboardPage`](https://github.com/perses/perses/blob/main/ui/e2e/src/pages/DashboardPage.ts) [page object model](https://playwright.dev/docs/pom) I configured for testing dashboards. It's basically an abstraction for selectors, helpers, and other bits like that for reuse across tests that hit the dashboard page in the app.

Reply:
And to make it a little more complex, `dashboardPage` is set up by the [`dashboardTest`](https://github.com/perses/perses/blob/main/ui/e2e/src/fixtures/dashboardTest.ts) [fixture](https://playwright.dev/docs/test-fixtures) I set up. The fixture helps encapsulate very common test setup and teardown, which in this case is "go to a specific dashboard" and some associated cleanup/teardown.